### ISSUE TITLE
Exception when ratelimits:hour is nil in Redis

### ISSUE THREAD

**Issue by apeckham** [myfreeweb/ring-ratelimit#5]
Using redis backend, when the `ratelimits:hour` key is nil (as it is when first deploying this middleware), there's an exception while parsing nil as an integer:

```
java.lang.NumberFormatException: null
	at java.lang.Integer.parseInt(Integer.java:542)
	at java.lang.Integer.parseInt(Integer.java:615)
	at ring.middleware.ratelimit.redis.RedisBackend.get_hour(redis.clj:14)
	at ring.middleware.ratelimit$wrap_ratelimit$fn__23314.invoke(ratelimit.clj:39)
```

Perhaps RedisBackend's `get-hour` should return nil when `(car/get hourname)` returns nil?

**Comment by apeckham** [comment]
Removed workaround from above because it seems to be resetting on every request... looking into it.

**Comment by apeckham** [comment]
Nevermind -- this was fixed in #3, but the released version (0.2.2) doesn't include that fix.
